### PR TITLE
CT-331 Add 'Day 6' notice

### PR DESCRIPTION
**Pull request by aliuk2012**
https://dsdmoj.atlassian.net/browse/CT-331
This notice will appear at the top of the case detail page for
every non-trigger FOI case.

We hope to eventually remove this notice once Private/Press Office
have access to the service and can view/audit FOI cases themselves
instead of relying on 'the grid'

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -64,7 +65,11 @@ en:
         state:
           accepted: Accept
           rejected: Reject
-
+  alerts:
+    escalation_warning_html: |
+      <strong>Important notice</strong><br>
+      This case in being reviewed by Press and Private office.
```
Comment (misaka) [inline config/locales/en.yml]: Small typo ... "This case *in* being reviewed ...", should be _is_ I guess?

File: app/views/cases/show.html.slim [hunk 2]
```diff
@@ -1,5 +1,8 @@
-= render partial: 'layouts/header', locals: {page_heading: 'Question'}
+- if @case.triggerable?
```
Comment (misaka) [inline app/views/cases/show.html.slim]: Is this the only criteria for showing this message? What if we're past the "Day 6" date?

Comment (aliuk2012) [inline app/views/cases/show.html.slim]: Good point, I thought oh just adding this in so its not displayed for other case types. Having spoken to Kellie she agrees that we should hide this if the day has gone past.

File: config/locales/en.yml (hunk 1 above)
Comment (aliuk2012) [inline config/locales/en.yml]: Good catch 👍

File: app/views/cases/show.html.slim [hunk 3]
```diff
@@ -1,5 +1,8 @@
-= render partial: 'layouts/header', locals: {page_heading: 'Question'}
+- if @case.triggerable? && Time.now < @case.escalation_deadline
```
Comment (EdwardAndress) [inline app/views/cases/show.html.slim]: Maybe we should have another method on Case that can be used here.

For example

def under_review?
  triggerable? && within_escalation_deadline?
end

def within_escalation_deadline?
  escalation_deadline.future? || escalation_deadline.present?
end

**Pull request merged**